app/services/ingest_iati_activities.rb

```diff
@@ -114,6 +114,9 @@ def call
       date = transaction_element.children.detect { |child| child.name.eql?("value") }.attributes["value-date"].value
       value = transaction_element.children.detect { |child| child.name.eql?("value") }.children.text
       transaction_type = transaction_element.children.detect { |child| child.name.eql?("transaction-type") }.attributes["code"].value
+      disbursement_channel = if transaction_element.children.detect { |child| child.name.eql?("disbursement-channel") }.present?
```

Comment:
I think it's good that we are trying to save this field if it's found. It would be good to have an assertion for this but I can't see one in any of our existing fixtures. Could we find a real activity on IATI with a `disbursement-channel` or fake one for testing purposes?